subprojects/distributions-dependencies/build.gradle.kts

```diff
@@ -27,7 +27,7 @@ val jaxbVersion = "3.0.0"
 val junit5Version = "5.8.2"
 val mavenVersion = "3.9.5"
 val mavenResolverVersion = "1.9.16" // Should remain in-sync with `mavenVersion`
-val nativePlatformVersion = "0.22-milestone-25"
+val nativePlatformVersion = "0.22-snapshot-20240226084139+0000"
```

Comment:
Can we use anything but a snapshot?

Reply:
Sorry, didn't realize this would trigger you. Converted back to a draft.